packages/react/src/components/Auth/Auth.tsx

```diff
@@ -1,5 +1,5 @@
 import { createStitches, createTheme } from '@stitches/core'
-import merge from 'just-merge'
+import merge from 'lodash/merge'
```

Comment:
This will not work as there is no package by that name. The package name is `lodash.merge`. And you will need to install the `@types/lodash.merge` for this to work in TS too.